### ISSUE TITLE
Vagrantfile synced_folder and provisioning script don't match

### ISSUE THREAD

**Issue by gergnz** [F5Networks/aws-deployments#12]
https://github.com/F5Networks/aws-deployments/blob/master/vagrant/Vagrantfile#L29
```
config.vm.synced_folder "../", "/aws-deployments"
```
https://github.com/F5Networks/aws-deployments/blob/master/build/scripts/setup_env_vbox.sh#L12
https://github.com/F5Networks/aws-deployments/blob/master/build/scripts/setup_env_vbox.sh#L23
```
cd aws-deployments
if ! egrep aws-deployments ~/.bash_profile ; then echo 'cd aws-deployments' >> ~/.bash_profile; fi
```

The line in the Vagrantfile is absolute "/", but the path in the provisioning script is relative to vagrant's home.

**Comment by gergnz** [comment]
Also it's broken in your .f5aws file.

**Comment by cmutzel** [comment]
Nice eyes.  Sorry for the mess.  I forgot to reload and test the vagrant environment itself. 
I've incorporate a fix into in commit 636a68b
